manual/src/reading.md

```diff
@@ -0,0 +1,104 @@
+# Reading From the Database
+
+## Database and Collection Handles
+
+Once you have a `Client`, you can call [`Client::database`](https://docs.rs/mongodb/latest/mongodb/struct.Client.html#method.database) to create a handle to a particular databse on the server, and [`Database::collection`](https://docs.rs/mongodb/latest/mongodb/struct.Database.html#method.collection) to create a handle to a particular collection in that database.  [`Database`](https://docs.rs/mongodb/latest/mongodb/struct.Database.html) and [`Collection`](https://docs.rs/mongodb/latest/mongodb/struct.Collection.html) handles are lightweight - creating them requires no IO, `clone`ing them is cheap, and they can be safely shared across threads or async tasks.  For example:
+```rust,no_run
+# extern crate mongodb;
+# extern crate tokio;
+# use mongodb::{bson::Document, Client, error::Result};
+# use tokio::task;
+#
+# async fn start_workers() -> Result<()> {
+# let client = Client::with_uri_str("mongodb://example.com").await?;
+let db = client.database("items");
+
+for i in 0..5 {
+    let db_ref = db.clone();
+
+    task::spawn(async move {
+        let collection = db_ref.collection::<Document>(&format!("coll{}", i));
+
+        // Do something with the collection
+    });
+}
+#
+# Ok(())
+# }
+```
+
+A `Collection` can be parameterized with any type that implements the `Serialize` and `Deserialize` traits from the [`serde`](https://serde.rs/) crate. This includes but is not limited to just `Document`. The various methods that accept or return instances of the documents in the collection will accept/return instances of the generic parameter (e.g. [`Collection::insert_one`](https://docs.rs/mongodb/latest/mongodb/struct.Collection.html#method.insert_one) accepts it as an argument, [`Collection::find_one`](https://docs.rs/mongodb/latest/mongodb/struct.Collection.html#method.find_one) returns an `Option` of it). It is recommended to define types that model your data which you can parameterize your `Collection`s with instead of `Document`, since doing so eliminates a lot of boilerplate deserialization code and is often more performant.
```

Comment:
technically `Serialize` and `Deserialize` don't both need to be implemented depending upon how the collection is being used. I don't think we need to get into the nitty gritty of that here, but can we update this language to reflect that the `T` for `Collection` doesn't have any specific requirements upon creation?

Reply:
Done.